### ISSUE TITLE
Make osx-arm64 builds available

### ISSUE THREAD

**Issue by corneliusroemer** [conda-forge/tsv-utils-feedstock#4]
Would be nice to have arm builds.

LDC doesn't have arm versions available yet, but we can pull the compiler directly from Github, something like this for build.sh:

```
if [ "$target_platform" == "osx-aarm64" ]; then
    wget https://github.com/ldc-developers/ldc/releases/download/v1.38.0/ldc2-1.38.0-osx-arm64.tar.xz
    tar -xf ldc2-1.38.0-osx-arm64.tar.xz
    export PATH=$PWD/ldc2-1.38.0-osx-arm64/bin:$PATH
    mkdir -p $PREFIX/lib
    cp ldc2-1.38.0-osx-arm64/lib/libdruntime-ldc-shared.*.*dylib $PREFIX/lib
    cp ldc2-1.38.0-osx-arm64/lib/libphobos2-ldc-shared.*.*dylib $PREFIX/lib
fi
```

**Comment by corneliusroemer** [comment]
Need to cross compile:
- adjust ldc2.conf to add info on where the osx-arm64 libs are
- download the osx-arm64 libs (via ldc2 compiled tarball)
- add necessary build flags

**Comment by corneliusroemer** [comment]
Resolved by #9
